src/ntfsck.c

```diff
@@ -2526,6 +2590,29 @@ static int ntfsck_scan_index_entries_btree(ntfs_volume *vol)
 		exit(-1);
 	}
 
+	/* find 'lost+found' directory in root directory */
+	lf_mftno = ntfs_inode_lookup_by_mbsname(ni, FILENAME_LOST_FOUND);
+	if (lf_mftno == (u64)-1) {
+		/* create 'lost+found' directory */
+		ucs_namelen = ntfs_mbstoucs(FILENAME_LOST_FOUND, &ucs_name);
+		if (ucs_namelen != -1) {
```

Comment:
WARNING: braces {} are not necessary for single statement blocks
#545: FILE: src/ntfsck.c:2598:
+		if (ucs_namelen != -1) {
+			lf_ni = ntfs_create(ni, 0, ucs_name, ucs_namelen, S_IFDIR);
+		}